.github/actions/generate-coverage/tests/test_scripts.py

```
@@ -709,6 +695,32 @@ def close(self) -> None:
     assert stderr.close_calls >= 1
 
 
+def test_run_cargo_passes_env_overrides(
+    monkeypatch: pytest.MonkeyPatch,
+) -> None:
+    """``_run_cargo`` merges env overrides into the spawned cargo process."""
+    mod = _load_module(monkeypatch, "run_rust")
+    monkeypatch.setattr(mod.os, "name", "nt")
+    monkeypatch.setattr(mod.typer, "echo", lambda *a, **k: None)
```

Ruff (0.15.10)

[warning] Unused lambda argument: `a`

(ARG005)